tests/utils/test_notifications.py

```diff
@@ -196,6 +197,170 @@ def test_condition_evaluation_timeout():
 
 
+# An example for a successful run inputs for webhook notifications
+run_example = {
+    "status": {
+        "notifications": {
+            "Test": {"status": "pending", "sent_time": None, "reason": None}
+        },
+        "results": {"return": 1},
+        "state": "completed",
+        "last_update": "2024-06-18T13:46:37.686443+00:00",
+        "start_time": "2024-06-18T13:46:37.392158+00:00",
+    },
+    "metadata": {
+        "uid": "b176e54e4ed24b28883aa69dce981601",
+        "project": "test-remote-workflow",
+        "name": "func-func",
+        "labels": {
+            "v3io_user": "admin",
+            "kind": "job",
+            "owner": "admin",
+            "mlrun/client_version": "1.7.0-rc21",
+            "mlrun/client_python_version": "3.9.18",
+            "host": "func-func-8lvl8",
+        },
+        "iteration": 0,
+    },
+    "spec": {
+        "function": "test-remote-workflow/func@8e0ddc3926470d5b97733679bb96738fa6dfd01b",
+        "parameters": {"x": 1},
+        "state_thresholds": {
+            "pending_scheduled": "1h",
+            "pending_not_scheduled": "-1",
+            "image_pull_backoff": "1h",
+            "executing": "24h",
+        },
+        "output_path": "v3io:///projects/test-remote-workflow/artifacts",
+        "notifications": [
+            {
+                "when": ["error", "completed"],
+                "name": "Test",
+                "params": {
+                    "url": "https://webhook.site/5da7ac4d-39dc-4896-b18f-e13c5712a96a",
+                    "method": "POST",
+                },
+                "message": "",
+                "status": "pending",
+                "condition": "",
+                "kind": "webhook",
+                "severity": "info",
+            }
+        ],
+        "handler": "func",
+    },
+}
+
+
+@pytest.mark.parametrize(
+    "override_body,runs",
+    [
+        ({"message": "runs: {{runs}}"}, run_example),
+        ({"message": "runs: {{ runs }}"}, run_example),
+        ({"message": "runs: {{runs}}"}, mlrun.model.RunObject.from_dict(run_example)),
+        (
+            {"message": "runs: {{ runs }}"},
+            mlrun.model.RunObject.from_dict(run_example),
+        ),
+    ],
+)
+async def test_webhook_overide_body_job_succeed(monkeypatch, override_body, runs):
+    requests_mock = _mock_async_response(monkeypatch, "post", {"id": "response-id"})
+
+    await WebhookNotification(
+        params={"override_body": override_body, "url": "http://test.com"}
+    ).push("test-message", "info", [runs])
+    expected_body = {
+        "message": "runs: [{'project': 'test-remote-workflow', 'name': 'func-func', 'host': 'func-func-8lvl8', "
+        "'status': {'state': 'completed', 'results': {'return': 1}}}]"
+    }
+    requests_mock.assert_called_once_with(
+        "http://test.com", headers={}, json=expected_body, ssl=None
+    )
+
+
+# An example for a failed runs inputs for webhook notifications
+run_example = {
```

Comment:
do you need both this and the one above?
please create a function that generates a run example and if you need to change values accept them as parameters

Reply:
fixed, i created one function that used for those two tests